tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -54,11 +54,10 @@ function main() {
   while true; do
     if [[ "$1" == "--nightly_flag" ]]; then
       NIGHTLY_BUILD=1
-    fi
-    if [[ "$1" == "--gpu" ]]; then
-      GPU_FLAG="--project_name tensorflow_gpu"
+    elif [[ "$1" == "--gpu" ]]; then
+      GPU_BUILD=1
     elif [[ "$1" == "--gpudirect" ]]; then
-      GPU_FLAG="--project_name tensorflow_gpudirect"
+      PKG_NAME_FLAG="--project_name tensorflow_gpudirect"
```

Comment:
http://ci.tensorflow.org/view/Experimental/job/experimental-gpu/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON2,label=gpu-linux/1/consoleFull

Testing it.